### PR TITLE
Fixed click availability and fee update when changing origin

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#86933mr3v
#86933qf9e

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-86933qf9e Android. XCM from swap screen. Fee is not recalculated after origin change](https://app.clickup.com/t/86933qf9e)

**Comment by stepanLav** [comment]
Task linked: [CU-86933mr3v Android. Can select source chain in xcms](https://app.clickup.com/t/86933mr3v)

File: feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/FeeLoaderMixin.kt [hunk 1]
```diff
@@ -211,6 +212,7 @@ fun <I1, I2, I3> FeeLoaderMixin.Presentation.connectWith(
     inputSource1: Flow<I1>,
     inputSource2: Flow<I2>,
     inputSource3: Flow<I3>,
+    invalidationSources: List<Flow<*>>,
```
Comment (valentunn) [inline feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/FeeLoaderMixin.kt:215]: additionalInvalidationSources

File: feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/FeeLoaderMixin.kt [hunk 2]
```diff
@@ -219,7 +221,8 @@ fun <I1, I2, I3> FeeLoaderMixin.Presentation.connectWith(
         inputSource1,
         inputSource2,
         inputSource3,
-    ) { input1, input2, input3 ->
+        invalidationSources.accumulate()
```
Comment (valentunn) [inline feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/FeeLoaderMixin.kt:224]: We can do `merge()` which is a little bit simpler

**Review by valentunn — APPROVED** [review]

**Review by valentunn — CHANGES_REQUESTED** [review]
I am imroving this PR since I've found that it is still possible for wrong fee to be displayed due to race conditions between `comissionFlowAsset` and fee calculation

**Review by valentunn — APPROVED** [review]

**Pull request merged**